x-pack/plugins/code/public/monaco/references/references_action.ts

```diff
@@ -18,7 +19,7 @@ export function registerReferencesAction(e: editor.IStandaloneCodeEditor) {
       const position = ed.getPosition();
       const { uri } = parseSchema(ed.getModel().uri.toString());
       const refUrl = `git:/${uri}!L${position.lineNumber - 1}:${position.column - 1}`;
-      const queries = queryString.parse(location.search);
+      const queries = url.parse(search, true).query;
```

Comment:
Is there a way to get the `search` from current url instead of passing by a parameter?

Reply:
`window.location.search` is empty if query string is after the hash.